docs/usage/config-presets.md

```diff
@@ -237,3 +198,44 @@ For example the result may be:
   "extends": ["local>myorgname/.github:renovate-config"]
 }
 ```
+
+## npm-hosted presets
```

Comment:
Question: should the `v25` branch have a commit which removes this section outright? I don't know if you can still use npm-hosted presets after `v25` lands?

Reply:
i think it's still supported. @rarkins Approve or close this pr 🙃

Reply:
Still supported in v25